I started today's session by googling the constructor error that I encountered two days ago.
aaaaaaand bingo, once again I tried to move forward too quickly and missed the important detail of passing in the aave lending pool address provider.
Now I have to figure out if the addresses are correct, as it doesn't seem that it's the case

I've narrowed down the problem to my deploy code. I have run my config through Patrick's deploy script, where he uses network.show_active, which I had forgotten to include in my deploy code.
Now the deploy script works well, and my wallet's public address gets printed in the console :
=================

```
brownie run scripts/deploy.py --network kovan
Brownie v1.17.2 - Python development framework for Ethereum

UpdatedVersionProject is the active project.

Running 'scripts/deploy.py::main'...
Transaction sent: 0x3b6480184d93af4a458e1422a38b97daf1d5fbdcd30ba30a768f7a816cf067f5
  Gas price: 2.425000008 gwei   Gas limit: 1189555   Nonce: 5
  Flashloan_logic.constructor confirmed   Block: 32331061   Gas used: 1081414 (90.91%)
  Flashloan_logic deployed at: 0x77a92b9a20D62942677eAEe9747973a51Da0909e

0x4a8e4183709bb59E0807D64E46cFe7747479d371
anne@AnneSager:~/Documents/FlashArbitrage/Flashloan_Arbitrage/UPDATED_VERSION$ brownie run scripts/deploy.py --network kovan
Brownie v1.17.2 - Python development framework for Ethereum

UpdatedVersionProject is the active project.

Running 'scripts/deploy.py::main'...
Transaction sent: 0x6ddd6b3071a4afe9028640f3dc3548c6c39148fd06e9b96b0ef00407df470056
  Gas price: 2.000000008 gwei   Gas limit: 1189555   Nonce: 6
  Flashloan_logic.constructor confirmed   Block: 32331089   Gas used: 1081414 (90.91%)
  Flashloan_logic deployed at: 0x8A8317Ed9B0d5F172eAbDD60083C3E7092eF87df

0x4a8e4183709bb59E0807D64E46cFe7747479d371
```

======================
